integrations/docker/images/stage-2/chip-build-nxp-zephyr/Dockerfile

```diff
@@ -10,23 +10,23 @@ RUN set -x \
 
 WORKDIR /opt/nxp-zephyr
 RUN set -x \
-    && wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.16.5/zephyr-sdk-0.16.5_linux-x86_64_minimal.tar.xz \
-    && tar xvf zephyr-sdk-0.16.5_linux-x86_64_minimal.tar.xz \
-    && rm -rf zephyr-sdk-0.16.5_linux-x86_64_minimal.tar.xz \
-    && zephyr-sdk-0.16.5/setup.sh -t arm-zephyr-eabi \
+    && wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.16.8/zephyr-sdk-0.16.8_linux-x86_64_minimal.tar.xz \
+    && tar xvf zephyr-sdk-0.16.8_linux-x86_64_minimal.tar.xz \
+    && rm -rf zephyr-sdk-0.16.8_linux-x86_64_minimal.tar.xz \
+    && zephyr-sdk-0.16.8/setup.sh -t arm-zephyr-eabi \
     && pip3 install --break-system-packages -U --no-cache-dir west \
-    && west init zephyrproject -m https://github.com/nxp-zephyr-ear/zephyr.git --mr zephyr_rw61x_v3.6_RFP \
-    && cd zephyrproject/zephyr \
+    && west init zephyrproject -m https://github.com/nxp-zephyr/nxp-zsdk.git --mr nxp-v3.7.0 \
+    && cd zephyrproject \
     && west update -o=--depth=1 -n \
     && west zephyr-export \
     && : # last line
 
 FROM ghcr.io/project-chip/chip-build:${VERSION}
 
-COPY --from=build /opt/nxp-zephyr/zephyr-sdk-0.16.5/ /opt/nxp-zephyr/zephyr-sdk-0.16.5/
+COPY --from=build /opt/nxp-zephyr/zephyr-sdk-0.16.8/ /opt/nxp-zephyr/zephyr-sdk-0.16.8/
 COPY --from=build /opt/nxp-zephyr/zephyrproject/ /opt/nxp-zephyr/zephyrproject/
 
 WORKDIR /opt/nxp-zephyr
 
 ENV ZEPHYR_NXP_BASE=/opt/nxp-zephyr/zephyrproject/zephyr
-ENV ZEPHYR_NXP_SDK_INSTALL_DIR=/opt/nxp-zephyr/zephyr-sdk-0.16.5
+ENV ZEPHYR_NXP_SDK_INSTALL_DIR=/opt/nxp-zephyr/zephyr-sdk-0.16.8
```

Comment:
This variable is also reflected in `integrations/docker/images/vscode/chip-build-vscode/Dockerfile`

Reply:
I will update it, thanks for catching it

Reply:
updated in last push, thanks !